integration-tests/foundation-grouped/pom.xml

```diff
@@ -96,12 +96,10 @@
             <groupId>org.apache.camel.quarkus</groupId>
             <artifactId>camel-quarkus-log</artifactId>
         </dependency>
-        <!-- https://github.com/apache/camel-quarkus/issues/4225
```

Comment:
camel-quarkus-microprofile-fault-tolerance should be kept out otherwise you'll keep getting the failure in foundation-grouped/pom.xml